src/util/differ.ts

```diff
@@ -1,26 +1,26 @@
 type ObjectJSON = Record<string, unknown>;
 type Field = string | number | ObjectJSON;
 
-class Differ {
-  isEmpty(value: Field): boolean {
-    if (value) {
-      return value === '';
-    }
-    return true;
+function isEmpty(value: Field): boolean {
+  if (value) {
+    return value === '';
```

Comment:
Use typescript tennary operator.
this if makes no empty string will always skip if..

Reply:
I managed to reduce the `isEmpty` function into this:
```js
const isEmpty = (value) => {
  return !value || value === '';
};
```
Any pointers about the ternary operator you mentioned ??